src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/ModFileInfo.java

```diff
@@ -115,12 +118,12 @@ public VersionRange getModLoaderVersion()
     }
 
     public Optional<Manifest> getManifest() {
-        Optional<Optional<Manifest>> manifest = this.manifest.get();
-        if (!manifest.isPresent()) {
-            manifest = Optional.of(modFile.getLocator().findManifest(modFile.getFilePath()));
-            this.manifest.set(manifest);
+        Optional<Manifest> result = manifest;
+        if (result == null) {
```

Comment:
Braces on new lines (might want to fix the one on the method signature as well while you're at it).